packages/lesswrong/components/posts/PostsItem2.tsx

```diff
@@ -45,6 +45,7 @@ export const styles = (theme) => ({
       flexWrap: "wrap",
       paddingTop: theme.spacing.unit,
       paddingBottom: theme.spacing.unit,
+      paddingLeft: 4
```

Comment:
Note: I experimented with both 4 and 8, and thought 8 looked a good bit better here. Did you check both?

Reply:
Actually, what I determined was that 5 was best according-to-me (8 looked weirdly big) but then forgot to port it over from the in-browser styling.